homeassistant/components/iron_os/switch.py

```diff
@@ -0,0 +1,163 @@
+"""Switch platform for IronOS integration."""
+
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+from enum import StrEnum
+from typing import Any
+
+from pynecil import CharSetting, SettingsDataResponse
+
+from homeassistant.components.switch import SwitchEntity, SwitchEntityDescription
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import IronOSConfigEntry
+from .coordinator import IronOSCoordinators
+from .entity import IronOSBaseEntity
+
+PARALLEL_UPDATES = 0
```

Comment:
can your library/pinecil deal with parallel requests (when switching)?

Reply:
the library already reads characteristics in parallel and didn't have any issues writing to characteristics in parallel